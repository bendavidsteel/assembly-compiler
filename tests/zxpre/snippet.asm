    mov %a,0x01     ; system call number (sys_exit)
    syscall         ; execute system call
