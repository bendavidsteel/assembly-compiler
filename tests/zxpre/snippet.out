         mov %a,0x01
         syscall
